### PR TITLE
1356 headers update

### PR DESCRIPTION
**Pull request by gpinkley**
Ticket_1356

Made my way through the following widgets changing to use of maxHeight to control header positioning, changing from custom components to standardized gemstone components where available, and updating outdated code:

AssetHistoryStats
AssetHistoryTable
EventSearchAssetVoltageDisturbances
EventSearchBreakerPerformance
EventSearchCapBankAnalyticOverview
EventSearchCorrelatedSags
EventSearchFaultSegments
EventSearchFileInfo
EventSearchNoteWindow
EventSearchOpenSEE 
EventSearchPQI
EventSearchRelayPerformance
InterruptionReport / HECCOIR
LineParameters
TVAESRIMap
TVAFaultInfo
TVALightningChart
TVALSC
TVAPQWeb
TVASIDA
TVASOE
TVAStructureInfo

### CODE REVIEW

File: SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchPreview/EventSearchAssetVoltageDisturbances.tsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ const EventSearchAssetVoltageDisturbances: React.FC<SEBrowser.IWidget<any>> = (p
     }
 
     return (
-        <div className="card">
+        <div className="card" style={{ maxHeight: props.maxHeight ?? 500, overflowY: 'auto' }}>
```
Comment (elwills) [inline SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchPreview/EventSearchAssetVoltageDisturbances.tsx:61]: This should be in the `tbodyStyle` property of the table, not the card. This makes the entire card scroll, including card and table headers.

File: SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchPreview/EventSearchCapBankAnalyticOverview.tsx [hunk 2]
```diff
@@ -24,75 +24,56 @@
...
+                        { key: 'PIS', field: 'PreInsertionSwitch', label: 'PreInsertionSwitching Condition' }
+                    ]}
+                    data={data}
+                    onClick={() => { }}
+                    onSort={() => { }}
+                    sortKey={''}
+                    ascending={true}
+                    tableClass="table"
+                    theadStyle={{ fontSize: 'smaller', display: 'table', tableLayout: 'fixed', width: '100%', height: 50 }}
+                    tbodyStyle={{ display: 'table', overflowY: 'scroll', width: '100%' }}
```
Comment (elwills) [inline SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchPreview/EventSearchCapBankAnalyticOverview.tsx:71]: `display` should be `'block'`, not `'table'` in order for the card to scroll correctly.

File: SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchPreview/EventSearchCorrelatedSags.tsx [hunk 3]
```diff
@@ -47,74 +60,46 @@ const EventSearchCorrelatedSags: React.FC<SEBrowser.IWidget<any>> = (props) => {
...
+                            { key: 'LineName', field: 'LineName', label: 'Line Name'}
+                        ]}
+                        data={data}
+                    onClick={() => { }}
+                    onSort={() => { }}
+                    sortKey={''}
+                    ascending={true}
+                    tableClass="table"
+                    theadStyle={{ fontSize: 'smaller', display: 'table', tableLayout: 'fixed', width: '100%', height: 50 }}
+                    tbodyStyle={{ display: 'table', overflowY: 'scroll', width: '100%', maxHeight: props.maxHeight ?? 500 }}
```
Comment (elwills) [inline SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchPreview/EventSearchCorrelatedSags.tsx:94]: `display` should be `'block'`, not `'table'`.

File: SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchPreview/EventSearchCorrelatedSags.tsx [hunk 4]
```diff
@@ -47,74 +60,46 @@ const EventSearchCorrelatedSags: React.FC<SEBrowser.IWidget<any>> = (props) => {
...
             <div className="card-header">Correlated Sags:</div>
 
             <div className="card-body">
-                <table className="table">
-                    <thead>
-                        <HeaderRow />
-                    </thead>
-                    <tbody>{tableRows}</tbody>
-                </table>
+            <div className="card-body" >
```
Comment (elwills) [inline SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchPreview/EventSearchCorrelatedSags.tsx:76]: This line is duplicated.

**Comment by clackner-gpa** [comment]
This is superceeded by #59

**Pull request closed without merging**